tests/Oryx.Integration.Tests/Php/PhpFpmConfigTest.cs

```diff
@@ -0,0 +1,170 @@
+        [InlineData("", "5", "", "2", "", "3", "", "1", null)] // defaults
+        [InlineData("false", null, "5", null, "5", null, "5", null, "pm.max_children must be a positive value")]
+        [InlineData("-1", null, "5", null, "5", null, "5", null, "pm.max_children must be a positive value")]
+        [InlineData("10", null, "5", null, "13", null, "12", null, "pm.min_spare_servers(12) and pm.max_spare_servers(13) cannot be greater than pm.max_children(10)")]
+        [InlineData("10", null, "9", null, "8", null, "7", null, "pm.start_servers(9) must not be less than pm.min_spare_servers(7) and not greater than pm.max_spare_servers(8)")]
+        public async Task PipelineTestInvocationsPhp74Async(
+            string fpmMaxChildren, string expectedFpmMaxChildren,
+            string fpmStartServers, string expectedFpmStartServers,
+            string fpmMaxSpareServers, string expectedFpmMaxSpareServers,
+            string fpmMinSpareServers, string expectedFpmMinSpareServers,
+            string failureOutputText)
+        {
+            await PhpFpmConfigTestAsync(
+                "7.4",
+                fpmMaxChildren, expectedFpmMaxChildren,
+                fpmStartServers, expectedFpmStartServers,
+                fpmMaxSpareServers, expectedFpmMaxSpareServers,
+                fpmMinSpareServers, expectedFpmMinSpareServers,
+                failureOutputText);
+        }
+
+        private async Task PhpFpmConfigTestAsync(
+            string phpVersion, 
+            string fpmMaxChildren, string expectedFpmMaxChildren,
+            string fpmStartServers, string expectedFpmStartServers,
+            string fpmMaxSpareServers, string expectedFpmMaxSpareServers,
+            string fpmMinSpareServers, string expectedFpmMinSpareServers,
+            string failureOutputText)
+        {
+            // Arrange
+            var appName = "php-fpm-config";
+            var hostDir = Path.Combine(_hostSamplesDir, "php", appName);
+            var volume = DockerVolume.CreateMirror(hostDir);
+            var appDir = volume.ContainerDir;
+            var appOutputDirVolume = CreateAppOutputDirVolume();
+            var appOutputDir = appOutputDirVolume.ContainerDir;
+
+            var buildScript = new ShellScriptBuilder()
+               .AddCommand($"oryx build {appDir} -i /tmp/int -o {appOutputDir} " +
+               $"--platform php --platform-version {phpVersion}")
+               .ToString();
+
+            var runScript = new ShellScriptBuilder()
+                .AddCommand($"oryx create-script -appPath {appOutputDir} -output {RunScriptPath} -bindPort {ContainerPort}")
+                .AddCommand("mkdir -p /home/site/wwwroot")
+                .AddCommand($"cp -rf {appOutputDir}/* /home/site/wwwroot")
+                .AddCommand(RunScriptPath)
+                .ToString();
+
+            var phpimageVersion = string.Concat(phpVersion, "-", "fpm");
+
+            if (failureOutputText == null)
```

Comment:
nit: I generally like to do `string.IsNullOrEmpty` for string checks, even if we explicitly set the value to `null` somewhere (helps keep consistent behavior if you were to change that `null` to `default` or `string.Empty`)